README.md

```diff
@@ -1 +1,15 @@
 # Sinatra Memo
+
+## Usage
+
+1. Install gems.
+```
+bundle install --path vendor/bundle
```

Comment:
こちら実行すると下記のような警告が表示されます。
```bash
[DEPRECATED] The `--path` flag is deprecated because it relies on being remembered across bundler invocations, which bundler will no longer do in future versions. Instead please use `bundle config set --local path 'vendor/bundle'`, and stop using this flag
```

よって、代わりに
```bash
bundle config set --local path 'vendor/bundle'
````
とするか、シンプルに
```bash
bundle install
```
にするのが良いと思います。